sdk/core/abort-controller/README.md

```diff
@@ -1,61 +1,67 @@
-# Azure Core Aborter library for JS
+# Azure Abort Controller library for JS
 
-`Aborter` is used by the Azure SDK for JavaScript to abort pending work based on timeouts or other signals. `Aborter` is compatible with the `AbortSignal` used by the `fetch` API built into modern browsers.
+`AborterController` is used by the Azure SDK for JavaScript to abort pending work based on timeouts or other signals.
+`AbortSignal` is compatible with the `AbortSignal` used by the `fetch` API built into modern browsers.
```

Comment:
Both AbortController and AbortSignal are built into browsers, to be clear!

Reply:
Updated documentation to hopefully be more clear on this point.